1.21.11 support/issue

Hey! There seems to be an issue when using Spigot 1.21.11:

```
Caused by: java.lang.ClassCastException: class net.minecraft.server.network.PlayerConnection cannot be cast to class io.netty.channel.Channel (net.minecraft.server.network.PlayerConnection and io.netty.channel.Channel are in unnamed module of loader java.net.URLClassLoader @6f496d9f)
	at me.matsubara.roulette.util.glowingentities.glowingentities.GlowingEntities$Packets.getChannel(GlowingEntities.java:692)
	at me.matsubara.roulette.util.glowingentities.glowingentities.GlowingEntities$Packets.addPacketsHandler(GlowingEntities.java:810)
	at me.matsubara.roulette.util.glowingentities.glowingentities.GlowingEntities.setGlowing(GlowingEntities.java:181)
	at me.matsubara.roulette.util.glowingentities.glowingentities.GlowingEntities.setGlowing(GlowingEntities.java:158)
	at me.matsubara.roulette.game.data.Bet.updateStandGlow(Bet.java:240)
	at me.matsubara.roulette.game.data.Bet.setStand(Bet.java:232)
	at me.matsubara.roulette.game.data.Bet.handle(Bet.java:168)
	at me.matsubara.roulette.game.Game.firstChipMove(Game.java:917)
	at me.matsubara.roulette.game.Game.playerBet(Game.java:1947)
	at me.matsubara.roulette.game.Game.takeMoneyAndPlaceBet(Game.java:1928)
	at me.matsubara.roulette.game.Game.newBet(Game.java:1906)
	at me.matsubara.roulette.gui.ChipGUI.handle(ChipGUI.java:232)
	at me.matsubara.roulette.listener.InventoryClick.onInventoryClick(InventoryClick.java:71)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:565)
	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:306)
	... 12 more
```
This is only specific to Spigot; it works perfectly fine with Paper. I've tried updating it myself, but I can't get the mappings generation to work. Any ideas? I know I can make the changes manually in `GlowingEntities.Packets#loadReflection`, but I think it would be better to be able to use the mappings

Thanks!

EDIT: This issue may also appear in [GuardianBeam](https://github.com/SkytAsul/GuardianBeam)

--- a/src/main/resources/fr/skytasul/glowingentities/mappings/spigot.txt
+++ b/src/main/resources/fr/skytasul/glowingentities/mappings/spigot.txt
@@ -1083,3 +1083,53 @@ net.minecraft.world.entity.Marker -> net.minecraft.world.entity.Marker:
 net.minecraft.network.protocol.Packet -> net.minecraft.network.protocol.Packet:
 net.minecraft.world.level.Level -> net.minecraft.world.level.World:
 net.minecraft.network.protocol.game.ClientboundSetPlayerTeamPacket$Parameters -> net.minecraft.network.protocol.game.PacketPlayOutScoreboardTeam$b:
+# reflection-remapper | 1.21.11
+net.minecraft.world.entity.EntityType -> net.minecraft.world.entity.EntityTypes:
+    MARKER -> aI
+    SHULKER -> bk
+net.minecraft.network.Connection -> net.minecraft.network.NetworkManager:
+    channel -> k
+net.minecraft.network.protocol.game.ClientboundSetPlayerTeamPacket -> net.minecraft.network.protocol.game.PacketPlayOutScoreboardTeam:
+net.minecraft.server.network.ServerCommonPacketListenerImpl -> net.minecraft.server.network.ServerCommonPacketListenerImpl:
+    connection -> e
+    send(net.minecraft.network.protocol.Packet) -> b
+net.minecraft.network.syncher.SynchedEntityData$DataValue -> net.minecraft.network.syncher.DataWatcher$c:
+    create(net.minecraft.network.syncher.EntityDataAccessor,java.lang.Object) -> a
+    id() -> a
+    serializer() -> b
+    value() -> c
+net.minecraft.server.level.ServerPlayer -> net.minecraft.server.level.EntityPlayer:
+    connection -> g
+net.minecraft.world.scores.Team$CollisionRule -> net.minecraft.world.scores.ScoreboardTeamBase$EnumTeamPush:
+    NEVER -> b
+net.minecraft.ChatFormatting -> net.minecraft.EnumChatFormat:
+    getByCode(char) -> a
+net.minecraft.network.syncher.EntityDataSerializer -> net.minecraft.network.syncher.DataWatcherSerializer:
+    createAccessor(int) -> a
+net.minecraft.network.syncher.SynchedEntityData$Builder -> net.minecraft.network.syncher.DataWatcher$a:
+    itemsById -> b
+    build() -> a
+net.minecraft.network.syncher.SyncedDataHolder -> net.minecraft.network.syncher.SyncedDataHolder:
+net.minecraft.network.protocol.game.ClientboundRemoveEntitiesPacket -> net.minecraft.network.protocol.game.PacketPlayOutEntityDestroy:
+net.minecraft.world.scores.PlayerTeam -> net.minecraft.world.scores.ScoreboardTeam:
+    setCollisionRule(net.minecraft.world.scores.Team$CollisionRule) -> a
+    setColor(net.minecraft.ChatFormatting) -> a
+net.minecraft.network.syncher.EntityDataAccessor -> net.minecraft.network.syncher.DataWatcherObject:
+net.minecraft.network.protocol.game.ClientboundAddEntityPacket -> net.minecraft.network.protocol.game.PacketPlayOutSpawnEntity:
+net.minecraft.network.protocol.BundlePacket -> net.minecraft.network.protocol.BundlePacket:
+    subPackets() -> b
+net.minecraft.network.syncher.SynchedEntityData -> net.minecraft.network.syncher.DataWatcher:
+    get(net.minecraft.network.syncher.EntityDataAccessor) -> a
+net.minecraft.world.scores.Scoreboard -> net.minecraft.world.scores.Scoreboard:
+net.minecraft.network.syncher.SynchedEntityData$DataItem -> net.minecraft.network.syncher.DataWatcher$Item:
+net.minecraft.world.entity.Entity -> net.minecraft.world.entity.Entity:
+    DATA_SHARED_FLAGS_ID -> aA
+    getEntityData() -> aD
+net.minecraft.network.protocol.game.ClientboundSetEntityDataPacket -> net.minecraft.network.protocol.game.PacketPlayOutEntityMetadata:
+    id -> c
+    packedItems -> d
+net.minecraft.world.phys.Vec3 -> net.minecraft.world.phys.Vec3D:
+net.minecraft.world.entity.Marker -> net.minecraft.world.entity.Marker:
+net.minecraft.network.protocol.Packet -> net.minecraft.network.protocol.Packet:
+net.minecraft.world.level.Level -> net.minecraft.world.level.World:
+net.minecraft.network.protocol.game.ClientboundSetPlayerTeamPacket$Parameters -> net.minecraft.network.protocol.game.PacketPlayOutScoreboardTeam$b:

--- a/src/mappingsgen/java/fr/skytasul/glowingentities/GlowingEntitiesMappingsGenerator.java
+++ b/src/mappingsgen/java/fr/skytasul/glowingentities/GlowingEntitiesMappingsGenerator.java
@@ -32,7 +32,7 @@ public class GlowingEntitiesMappingsGenerator {
 				"1.18", "1.18.1", "1.18.2",
 				"1.19", "1.19.1", "1.19.2", "1.19.3", "1.19.4",
 				"1.20.1", "1.20.2", "1.20.4", "1.20.6",
-				"1.21", "1.21.1", "1.21.3", "1.21.4", "1.21.5", "1.21.6", "1.21.7", "1.21.8", "1.21.9", "1.21.10")) {
+				"1.21", "1.21.1", "1.21.3", "1.21.4", "1.21.5", "1.21.6", "1.21.7", "1.21.8", "1.21.9", "1.21.10", "1.21.11")) {
 			try {
 				LOGGER.info("Downloading mappings for " + version + "...");
 				var minecraftMappings = mappingsProvider.loadMinecraftMappings(version);

